### PR TITLE
Restrict judging to users that have synced devpost

### PR DESCRIPTION
**Pull request by ianmah**

### CODE REVIEW

File: src/containers/SubmissionLink.js [hunk 1]
```diff
@@ -13,17 +13,9 @@ const getProjectByEmail = async email => {
...
   const [email, setEmail] = useState('')
   const [message, setMessage] = useState('')
 
-  useEffect(() => {
-    const unsubscribe = getLivesiteDoc(livesiteDoc =>
-      setIsSubmissionsOpen(livesiteDoc.submissionsOpen)
-    )
-    return unsubscribe
-  }, [setIsSubmissionsOpen])
-
```
Comment (jackyzha0) [inline src/containers/SubmissionLink.js:26]: 🧼

File: src/pages/Judging/index.js [hunk 2]
```diff
@@ -85,6 +95,19 @@ export default () => {
     return <Loading />
   }
 
+  if (isBlocked) {
+    return (
+      <HeroPage>
+        <h2>Error, permission denied</h2>
```
Comment (jackyzha0) [inline src/pages/Judging/index.js:101]: can we use H2 here

File: src/pages/Judging/index.js [hunk 3]
```diff
@@ -69,13 +71,21 @@ const getProjects = async userId => {
...
   const [projects, setProjects] = useState([])
 
   useEffect(() => {
     ;(async () => setProjects(await getProjects(user.uid)))()
   }, [setProjects, user.uid])
 
+  useEffect(() => {
+    ;(async () => {
+      const applicantData = (await applicantsRef.doc(user.uid).get()).data()
+      setIsBlocked(!!!applicantData.submittedProject)
```
Comment (jackyzha0) [inline src/pages/Judging/index.js:85]: i think ! === !!!

File: src/pages/Submission.js [hunk 4]
```diff
@@ -47,8 +47,8 @@ export default () => {
   if (!isSubmissionsOpen) {
     return (
       <HeroPage>
-        <h2>Submissions are not open.</h2>
-        <h2>Check back here later to submit your project!</h2>
+        <h2>Submissions are not open</h2>
+        <p>Check back here later</p>
```
Comment (jackyzha0) [inline src/pages/Submission.js:51]: can we change this to H2 and P

File: src/pages/Judging/index.js (hunk 3 above)
Comment (ianmah) [reply src/pages/Judging/index.js]: you've seen `!`, now get ready for `!!!!!`

**Review by jackyzha0 — APPROVED** [review]
nice

**Pull request merged**